doc/usage/extensions/autosummary.rst

```diff
@@ -303,6 +303,27 @@ The following variables are available in the templates:
 
    .. versionadded:: 1.8.0
 
+.. data:: inherited_methods
+
+   List containing names of only inherited methods.  Only available for
+   classes.
+
+   .. versionadded:: 7.2.7
+
+.. data:: inherited_attributes
+
+   List containing names of only inherited attributes.  Only available for
+   classes.
+
+   .. versionadded:: 7.2.7
+
+.. data:: inherited_paths
```

Comment:
```suggestion
.. data:: inherited_qualname
```

We usually refer this path as the fully qualified name or sometimes the canonical name. We use `:canonical:` in Sphinx, so you could also have `canonical_inherited_names`.

Reply:
Have adopeted qualnames